docs/migration/mui_migration_1.0.md

```diff
@@ -0,0 +1,15 @@
+# Consumption guide of shared components
+
+## Breaking changes
```

Comment:
to the breaking changes of which major version is this documentation referring to? Please enhance.